srcCxx/ShapeType_main.cxx

```diff
@@ -54,8 +65,50 @@ void setup_signal_handler()
     signal(SIGABRT, signal_handler); //Abort
 }
 
+/*************************************************************/
+enum Verbosity
+{
+    ERROR=1,
+    DEBUG=2,
+};
+
+class Logger{
+public:
```

Comment:
**Note to self**: [dds-rtps reference](https://github.com/omg-dds/dds-rtps/blob/c68ff26f96a41d1544fc9cd5564ed275ef520884/srcCxx/shape_main.cxx#L176)